mkdocs/docs/release_notes/version_0.16.0.md

```diff
@@ -0,0 +1,103 @@
+# dymos 0.16.0
+
+## Backwards incompatible changes
+
+There is no longer a distinction between input parameters and design parameters.
+While the add_input_parameter and add_design_paramaters will still function until dymos 1.0.0, the path to the parameters themselves is changed in a backwards incompatible way.
+
+{{ upgrade_doc('dymos.test.test_upgrade_guide.TestUpgrade_0_16_0.test_parameters',
+               feature='set_val',
+               old_label='dymos < 0.16.0',
+               new_label='dymos >= 0.16.0') }}
+
+Likewise, the path to the parameter values in the timeseries outputs is changed:
+
+{{ upgrade_doc('dymos.test.test_upgrade_guide.TestUpgrade_0_16_0.test_parameters',
+               feature='parameter_timeseries',
+               old_label='dymos < 0.16.0',
+               new_label='dymos >= 0.16.0') }}
+
+## Enhancements
+
+### Omitting parameter values from a timeseries
+
+Since parameters are static, including their values in the timeseries can lead to more data being stored than necessary.
+If desired, one can avoid storing parameters in timeseries using the `include_timeseries=False` directive.
+
+{{ upgrade_doc('dymos.test.test_upgrade_guide.TestUpgrade_0_16_0.test_parameter_no_include_timeseries',
+               feature='parameter_no_timeseries',
+               old_label='dymos < 0.16.0',
+               new_label='dymos >= 0.16.0') }}
+
+### Timeseries improvements
+
+In previous dymos versions, when adding ODE outputs to a timeseries, the user was required to specify the units and shape of the output variable.
+This is redundant since that information was defined in the timeseries itself.
+However, dymos had no way to determine that variable metadata.
+With recent changes to OpenMDAO, this is now possible and the addition of ODE variables is simplified by no longer having to specify units or shape of the timeseries outputs.
+
+=== "Adding timseries otuputs in dymos < 0.16.0"
+    ```
+    phase.add_timeseries_output('aero.lift_force', units='N', shape=(1,))
+    ```
```

Comment:
This markdown is cool and easy, but one thing it is missing is testing. The one we use in Sphinx pulls a block of code out of an existing test using a couple of search strings to find the start and end line (just for the "good" column though; the bad/old column is just a block of untested code like here).  It's a little harder on readability of the rst file perhaps, but it has the advantage that any changes to the API will get automatically reflected here.  Granted, api changes don't happen that often anymore.

Reply:
(note, this was a general comment, won't hold up the review for it.)

Reply:
Agreed.  I wrote these docs before this capability was merged in.  Now that it's present I'll do it the right way.